ibis/backends/base/sql/__init__.py

```diff
@@ -198,6 +196,12 @@ def to_pyarrow_batches(
 
         return pa.ipc.RecordBatchReader.from_batches(schema.to_pyarrow(), batches)
 
+    def _compile_udfs(self, expr: ir.Expr):
+        pass
+
+    def _register_udfs(self, expr: ir.Expr):
+        pass
```

Comment:
Should these raise a `NotImplementedError` (or fallback to the above) rather than a silent `pass`? Won't this silently pass through for backends that don't support UDFs?

Also could use a docstring on both of these.

Reply:
Added docstrings, and now raising `NotImplementedError`.